### PR TITLE
Cmake fix

### PR DESCRIPTION
**Pull request by DethRaid**
- Only require CMake 3.10, not 3.13 
- Make AppVeyor yell at us when builds fail

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -185,7 +185,8 @@ set(3RD_PARTY_SOURCE
 ##########################################
 if (UNIX)
     add_compile_options_if_supported(-g -fno-omit-frame-pointer -fPIC -fdiagnostics-color=always)
-    add_link_options(-fno-omit-frame-pointer)
+    # TODO: Find a CMake 3.12-compatible way to do this
+    # add_link_options(-fno-omit-frame-pointer)
```
Comment (cwfitzgerald) [inline CMakeLists.txt]: This should be on the command line, not just on the link line. This effects codegen.

File: appveyor.yml [hunk 2]
```diff
@@ -36,3 +36,11 @@ build:
...
 - cmd: dir C:\projects\nova-renderer\build\%CONFIGURATION%\
 - cmd: C:\projects\nova-renderer\build\%CONFIGURATION%\nova-test-unit.exe
+
+on_success:
+  - ps: Invoke-RestMethod https://raw.githubusercontent.com/DiscordHooks/appveyor-discord-webhook/master/send.ps1 -o send.ps1
+  - ps: ./send.ps1 success $env:WEBHOOK_URL
+on_failure:
+  - ps: Invoke-RestMethod https://raw.githubusercontent.com/DiscordHooks/appveyor-discord-webhook/master/send.ps1 -o send.ps1
+  - ps: ./send.ps1 failure $env:WEBHOOK_URL
+
```
Comment (cwfitzgerald) [inline appveyor.yml]: End of line issues

**Review by cwfitzgerald — CHANGES_REQUESTED** [review]

File: appveyor.yml (hunk 2 above)
Comment (DethRaid) [reply appveyor.yml]: Fixed

**Comment by cwfitzgerald** [comment]
Test post please ignore

**Comment by Janrupf** [comment]
Looks good now, waiting for @cwfitzgerald to approve changes

**Review by cwfitzgerald — APPROVED** [review]

**Pull request merged**